src/libratbag.h

```diff
@@ -1300,6 +1322,139 @@ enum ratbag_button_action_special
 ratbag_button_get_special(struct ratbag_button *button);
 
 /**
+ * @ingroup led
+ *
+ * LED mods.
```

Comment:
typo, but this would also need a bit more of an explanation for each mode. on/off is obvious, but cycle for example isn't clear.